stdlib/bool.mli

```diff
@@ -39,6 +39,18 @@ external ( || ) : bool -> bool -> bool = "%sequor"
     If [e0] evaluates to [true], [e1] is not evaluated. Right-associative
     operator at precedence level 2/11. *)
 
+val logand : bool -> bool -> bool
+(** [logand e1 e2] is the eager boolean conjunction of expressions [e1] and
+    [e2]. *)
```

Comment:
Just a suggestion but I would prefer a more "concrete" docstring:
```suggestion
(** [logand b1 b2] is [true] if and only if [b1] and [b2] are both [true]. *)
```
In particular, I think using "e1", "e2", "expression", ... is a bit misleading here; I would prefer "b1", "b2", "boolean", ... instead.